test/examples_test.go

```diff
@@ -173,8 +175,8 @@ func validateObject(obj runtime.Object) (errors field.ErrorList) {
 			t.Namespace = api.NamespaceDefault
 		}
 		errors = ext_validation.ValidateIngress(t)
-	case *extensions.PodSecurityPolicy:
-		errors = ext_validation.ValidatePodSecurityPolicy(t)
+	case *policyv1beta1.PodSecurityPolicy:
```

Comment:
why is this an external type? none of the others are...

Reply:
Because PSP from policy group has only external type.

Reply:
same is true for Deployments in the apps API group, and yet that is handled as an extensions.Deployment internally

https://github.com/kubernetes/website/blob/master/docs/concepts/overview/working-with-objects/nginx-deployment.yaml

https://github.com/kubernetes/website/pull/7562/files#diff-1b1c44245582a644b62a97397e32591dR314

Reply:
Indeed, the conversion from policy group in YAML file and extensions in a test, works without any changes. Thanks! Updated.